common/metacity-1/button-border.svg

```diff
@@ -1,4 +1,5 @@
-﻿<svg viewBox="0 0 16 16">
+﻿<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
     <g id="titlebutton-max-hover" transform="translate(-513,185.63782)" inkscape:label="#g6284">

```

Comment:
My SVG's are perfectly viewable, even without the `<?xml ?>`-tag and `xmlns`-attribute.

And these have been tested on my system well before submitting this PR. I added the socalled *mandatory* tags merely to follow a certain standard.

The `inkscape:label` was already there, if you look at the code for one second. But I will make another commit just to remove that attribute which does nothing but create a label inside some vector editor software.